sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py

```diff
@@ -96,3 +97,36 @@ def __repr__(self):
         return "EventGridEvent(subject={}, event_type={}, id={}, event_time={})".format(
             self.subject, self.event_type, self.id, self.event_time
         )[:1024]
+
+    @classmethod
+    def from_dict(cls, data, key_extractors=None, content_type=None):
+        event = EventGridEvent._get_bytes(data)
+        super(EventGridEvent, cls).from_dict(event, key_extractors, content_type)
+
+    @staticmethod
+    def _get_bytes(obj):
```

Comment:
Same comment re name and making a separate utility function.

Reply:
Shouldn't this method now be removed?

Reply:
oops - removing